README.md

```diff
@@ -1,164 +1,188 @@
+To use the the official installer to install, configure, and run Discovery:
 
-You must be running the following commands as root to install the discovery installer via the RPM:
-
-```
-# subscription-manager repos --enable discovery-1-for-rhel-9-x86_64-rpms
-# dnf install -y discovery-installer
+```sh
+sudo subscription-manager repos --enable discovery-1-for-rhel-9-x86_64-rpms
+sudo dnf install -y discovery-installer
+discovery-installer install
+podman login registry.redhat.io  # REQUIRED before starting discovery-app
+systemctl --user start discovery-app
 ```
 
-```
-# type discovery-installer
-discovery-installer is /usr/bin/discovery-installer
+A few seconds later, you may access Discovery on https://localhost:9443
+
+If you want to access Discovery from systems outside of localhost, you may need to add a rule to allow access through the firewall:
+
+```sh
+sudo firewall-cmd --permanent --add-port=9443/tcp  # optional if you want external access
+sudo firewall-cmd --reload  # optional if you want external access
 ```
 
-Discovery must then be installed as a regular (non-root) user in the system. For the following examples we will use the username **discovery**
+## Running from source (developers only)
 
-Log in to the _discovery_ user account then install Discovery:
+Alternatively, if you are a Discovery developer and need to run this repo from source:
 
+```sh
+git clone https://github.com/quipucords/discovery-quadlets.git discovery-quadlets
+cd discovery-quadlets
+DISCOVERY_PKG_ROOT=$(pwd) ./bin/discovery-installer install
+podman login registry.redhat.io  # REQUIRED before starting discovery-app
+systemctl --user start discovery-app
 ```
-discovery $ pwd
-/home/discovery
-discovery $ type discovery-installer
-discovery-installer is /usr/bin/discovery-installer
-discovery $ discovery-installer
-Usage: discovery-installer [-v | --verbose] <command>
 
-Where <command> is one of:
-  install                  - Install Discovery
-  uninstall                - Uninstall Discovery
-  create-server-password   - Create a Discovery server password
-  create-app-secret        - Create a Discovery application secret
-```
+A few seconds (or minutes depending on your hardware) later, you may access Discovery on https://localhost:9443
 
-## Discovery Pre-requisites
+## Accessing Discovery
 
-### Podman Secrets
+You can access Discovery on the host where it was installed at https://localhost:9443.
 
-Before starting Discovery you need to create the following _podman_ secrets:
+If that host has a custom hostname, such as `my-discovery-server`, and you have added appropriate firewall rules, you may be able to access Discovery from other systems on your network at https://my-discovery-server:9443 or https://my-discovery-server.local:9443. Further hostname, DNS, firewall, and network configuration instaructions are outside the scope of this document and not officially supported.
```

Comment:
s/instaructions/instructions/

Reply:
Thanks! I ran the document through a spell-checker after you noticed that, and I found one other typo too. Fixed and pushed. 👍